ubuntudesign/www.ubuntu.com
# 1610 fw desktop

## Done

Design review amends on full width desktop section

## QA

Check the desktop section in s/m/l

Awaiting final design sign off and there are still a few more amends to make

## Issue / Card

Card: https://trello.com/c/wcj7FtDU
